packages/postcss-config/package.json

```diff
@@ -26,17 +26,14 @@
     "prepack": "node ../../tools/clean-package-json.mjs"
   },
   "dependencies": {
-    "@types/postcss-mixins": "^9.0.6",
     "postcss-calc": "^10.1.1",
     "postcss-mixins": "^11.0.3",
-    "postcss-preset-env": "~7.8.3"
+    "postcss-preset-env": "^7.8.3"
   },
   "devDependencies": {
+    "@types/postcss-mixins": "^9.0.6",
     "postcss": "^8.5.3"
```

Comment:
Should this have TypeScript as a dev dependency or should we remove that from the other two for consistency.

Reply:
@101100 `@pandell/eslint-config` has to include TypeScript because of `typescript-eslint` peer dependency requirement. As for `@pandell/webpack-config`, I have been waiting for feedback from your JV experiment - it TypeScript is not needed, we should remove it.

Reply:
Why do we need the TypeScript peer dependency in `@pandell/eslint-config`?

Reply:
Or another way to put it, aren't we saying "You probably want 'eslint' if you are using this package." when really it isn't required to use the package. So if we are going minimal, we should be able to remove the peer dependencies completely, right?

Reply:
@101100 `@pandell/eslint-config` only has `typescript` as an optional peer dependency (i.e. not enforced by `yarn`), and it is there because we enable TypeScript rules by default, so chances are you will need TypeScript if you are adding `@pandell/eslint-config`.